style.css

```diff
@@ -510,8 +1105,611 @@ body {
+    -ms-flex-align: center;
+    align-items: center;
+    -webkit-box-flex: 2;
+    -ms-flex: 2;
+    flex: 2;
+    height: 100%;
+    margin: 0;
+  }
+
+  .middle-section__img {
+    height: 100%;
+    max-width: 648px;
+    margin: 0;
+  }
+
+  .middle-section__clutch {
+    -webkit-box-flex: 1;
+    -ms-flex: 1;
+    flex: 1;
+    margin-top: 0;
+    margin-left: 40px;
+    margin-right: 0;
+  }
+
+  .middle-section__desription {
+    opacity: .5;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section {
+    padding: 0 40px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+  }
+
+  .bottom-section__item {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    margin-left: -10px;
+  }
+
+  .bottom-section__btn:hover {
+    background-color: #fff;
+  }
+
+  .bottom-section__plus,
+  .bottom-section__minus {
+    border: none;
+    margin-bottom: 50px;
+  }
+
+  .bottom-section__plus:hover {
+    transform: scale(1.2);
+    cursor: pointer;
+  }
+
+  .bottom-section__minus:hover {
+    transform: scale(1.2);
+    cursor: pointer;
+  }
+
+  .bottom-section__link:hover {
+    transform: scale(1.05);
+  }
+
+  .bottom-section__value {
+    margin: 0 10px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+  }
+}
+
+@media (min-width: 410px) {
+  .bottom-section__text-block {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+  }
+}
+
+@media (min-width: 520px) {
+  .bottom-section__text-block {
+    margin-left: 25%;
+    margin-right: 20%;
+  }
+
+  .bottom-section__image {
+    padding-left: 35px;
+    padding-right: 30px;
+  }
+}
+
+@media (min-width: 768px) {
+  .bottom-section__block {
+    margin-left: 30%;
+    margin-right: 15%;
+  }
+
+  .bottom-section__image {
+    padding-left: 45px;
+    padding-right: 40px;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section__block {
+    position: absolute;
+    top: 30%;
+    right: 3%;
+    padding: 30px 40px;
+    margin-left: 50%;
+    margin-right: 0;
+  }
+
+  .bottom-section__image {
+    padding: 0;
+    margin-right: 40%;
+  }
+
+  .bottom-section__img {
+    margin: 137px 0;
+  }
+
+  .bottom-section__text-block {
+    -webkit-box-align: start;
+    -ms-flex-align: start;
+    align-items: flex-start;
+  }
+}
+
+@media (min-width: 1200px) {
+  .bottom-section__block {
+    margin-right: 7%;
+  }
+
+  .bottom-section__image {
+    margin-left: 3%;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section__title,
+  .bottom-section__subtitle,
+  .bottom-section__description {
+    margin-left: -80px;
+  }
+
+  .bottom-section__title {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 41px;
+    color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)